### PR TITLE
determine_molecules: bug fix

### PR DESCRIPTION
**Pull request by afshinfard**
fixed a bug

### CODE REVIEW

**Comment by afshinfard** [comment]
The failed check seem to be for `physlr_paf_metrics()` (not in my edits for the bug fix) and caused by unused variables in that function. It's a bit tricky and actually those variables are kinda used. They seemingly need to bring AI into `pylint` to distinguish such cases :))

**Comment by lcoombe** [comment]
@afshinfard - I just fixed the pylint error in #71 :)

When we have that merged into master, you could rebase your branch on master and then azure should be happy.

**Comment by afshinfard** [comment]
Thanks @lcoombe, I'll rebase tomorrow.

File: data/Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ physlr_path=$(shell dirname $$(dirname $(realpath $(MAKEFILE_LIST))))
 bin=$(physlr_path)/bin
 
 # Python interpreter.
-python=env PYTHONPATH=$(physlr_path) pypy3
+python=env PYTHONPATH=$(physlr_path) /projects/btl/aafshinfard/virtuEnv/pypy3/bin/pypy3
```
Comment (sjackman) [inline data/Makefile]: Please revert this line.

**Pull request closed without merging**